gapic/src/main/com/google/gapid/models/Profile.java

```diff
@@ -204,7 +204,7 @@ public TimeSpan getSlicesTimeSpan() {
 
     public Double getGpuPerformance(List<Long> commandIndex, int metricId) {
       CommandIndex indexStr = new CommandIndex(commandIndex);
-      if (!perfLookup.containsKey(indexStr)) {
+      if (!perfLookup.containsKey(indexStr) || !perfLookup.get(indexStr).containsKey(metricId)) {
```

Comment:
I would prefer
```
Map<Integer, Double> metrics = perfLookup.get(indexStr);
return (metrics == null) ? Double.NaN : metrics.getOrDefault(metricId, Double.NaN);
```

Reply:
Thanks for suggestion, make sense to improve the efficiency here since it would be run frequently, done with the updated branch!